ads/uas.md

```diff
@@ -0,0 +1,76 @@
+<!---
```

Comment:
Please add link to your documentation from `amp-ad.md`

Reply:
done